src/main/java/com/farao_community/farao/gridcapa/job_launcher/JobLauncherService.java

```diff
@@ -85,6 +93,25 @@ public boolean launchJob(String timestamp) {
         return false;
     }
 
+    TaskDto replaceFilenameWithPresignedUrls(TaskDto taskDto) {
+        List<ProcessFileDto> newInput = new ArrayList<>();
+        for (ProcessFileDto input : taskDto.getInputs()) {
+            ProcessFileDto processFileDto = new ProcessFileDto(input.getFileType(),
+                    input.getProcessFileStatus(),
+                    input.getFilename(),
+                    input.getLastModificationDate(),
+                    minioAdapter.generatePreSignedUrlFromFullMinioPath(input.getFileUrl(), 1));
```

Comment:
That's working but I don't think it's clean to modify ProcessFileDto here. This is also linked with my comment in the task manager PR, it's not pretty to use the same variable for two different meaning. In the task manager the FileUrl correspond to the object key (Minio path) and here is replaced by an url. I think also it's better to use the minio.generatePresignedUrlFromFullMinioPath method in the process adapter when it creates the process request. We can discuss about it !